src/__tests__/posthog-core.ts

```diff
@@ -825,7 +842,10 @@ describe('posthog core', () => {
             posthog.sessionRecording = {
                 afterDecideResponse: jest.fn(),
                 startIfEnabledOrStop: jest.fn(),
-            } as unknown as SessionRecording
+            } as OnlyValidKeys<
```

Comment:
interesting... i expected this change to highlight the incorrect setup here that the `as unknown` is hiding